src-java/floodlight-service/floodlight-modules/src/test/java/org/openkilda/floodlight/converter/OfFlowStatsMapperTest.java

```diff
@@ -135,29 +136,29 @@ public void testFlowEntry() {
         assertEquals(udpSrc.toString(), entry.getMatch().getUdpSrc());
         assertEquals(udpDst.toString(), entry.getMatch().getUdpDst());
 
-        FlowSetFieldAction flowSetEthSrcAction = new FlowSetFieldAction("eth_src", MAC_ADDRESS_1);
-        FlowSetFieldAction flowSetEthDstAction = new FlowSetFieldAction("eth_dst", MAC_ADDRESS_2);
-        FlowCopyFieldAction flowCopyFieldAction = FlowCopyFieldAction.builder()
-                .bits(String.valueOf(bits))
-                .srcOffset(String.valueOf(srcOffset))
-                .dstOffset(String.valueOf(dstOffset))
-                .srcOxm(String.valueOf(oxmSrcHeader))
-                .dstOxm(String.valueOf(oxmDstHeader))
-                .build();
-        FlowSwapFieldAction flowSwapFieldAction = FlowSwapFieldAction.builder()
-                .bits(String.valueOf(bits))
-                .srcOffset(String.valueOf(srcOffset))
-                .dstOffset(String.valueOf(dstOffset))
-                .srcOxm(String.valueOf(oxmSrcHeader))
-                .dstOxm(String.valueOf(oxmDstHeader))
-                .build();
-        FlowApplyActions applyActions = new FlowApplyActions(port.toString(),
-                Lists.newArrayList(flowSetEthSrcAction, flowSetEthDstAction), ethType.toString(), null, null, null,
-                group.toString(), flowCopyFieldAction, flowSwapFieldAction);
+
+        FlowApplyActions applyActions = buildFlowApplyActions();
         FlowInstructions instructions = new FlowInstructions(applyActions, null, meterId, goToTable.getValue());
         assertEquals(instructions, entry.getInstructions());
     }
 
+    @Test
+    public void toFlowInstructions() {
+        FlowApplyActions applyActions = buildFlowApplyActions();
+        FlowInstructions expectedNoneEmptyFlowInstructions = new FlowInstructions(applyActions,
+                null, meterId, goToTable.getValue());
+        FlowInstructions actualNoneEmptyFlowInstructions
+                = OfFlowStatsMapper.INSTANCE.toFlowInstructions(buildInstruction());
+
+        assertEquals(expectedNoneEmptyFlowInstructions, actualNoneEmptyFlowInstructions);
+
+        FlowInstructions expectedEmptyFlowInstructions = FlowInstructions.builder().none(NONE_DROP).build();
+        List<OFInstruction> emptyInstructions = Collections.emptyList();
+        FlowInstructions actualEmptyInstructions = OfFlowStatsMapper.INSTANCE.toFlowInstructions(emptyInstructions);
+
+        assertEquals(expectedEmptyFlowInstructions, actualEmptyInstructions);
+    }
```

Comment:
Here we have 2 test data for 2 test cases. Please split this test into 2 tests - one with some flow instructions and one with empty instructions

Reply:
thanks, fixed